cel/README.md

```diff
@@ -5,25 +5,33 @@
 
 This is an experimental project that provides support for Common Expression Language (CEL) in Tekton Pipelines.
 The functionality is provided by a controller that implements the Custom Task interface. Its use cases include 
-evaluating complex expressions to be used in [`WhenExpressions`](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#guard-task-execution-using-whenexpressions)
-in subsequent `Tasks` to guard their execution. 
+evaluating complex expressions to be used in [`when` expressions][when-expressions] in subsequent `Tasks` to guard 
+their execution. 
```

Comment:
in some ways this probably goes without saying (it's true for all the experimental custom tasks) but i think it might be worth adding a word of caution expressing that we do have a desire to make this a top level feature, but at that point we'll need to re-examine the design via TEP and it might change

Reply:
👍🏾 added the disclaimer